blog/2023-07-07-hidden-costs-ofmonitoring/index.md

```diff
@@ -0,0 +1,227 @@
+---
+slug: hidden-costs-of-monitoring
+title: "The Hidden Costs of Monitoring"
+authors: satya
+tags: [netdata-monitoring, costs-of-monitoring,hidden-costs,data-costs,opportunity-costs,retention-costs,cost-of-delayed-rca]
+keywords: [netdata-monitoring, costs-of-monitoring,hidden-costs,data-costs,opportunity-costs,retention-costs,cost-of-delayed-rca]
+image: ./img/hidden-costs.png
+---
+
+![stacked-netdata](./img/netdata-parents.png)
```

Comment:
We don't need this image do we?

Reply:
These are the templates @christophidesp and @Ancairon came up with: https://www.canva.com/design/DAFj7VB47wk/UTD23wJTKcdd2uJLHydhYw/edit?utm_content=DAFj7VB47wk&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

Reply:
Oops I had the wrong image.. Fixed it.